cmd/get.go

```diff
@@ -27,11 +28,21 @@ import (
 )
 
 func get(cmd *cobra.Command, args []string) (err error) {
+	if len(args) < 1 {
+		return errors.New("source is required")
+	}
+
 	src, err := validatePath(args[0])
-	dst := args[1]
+	if err != nil {
+		return
+	}
 
-	if dst == "" {
+	var dst string
+	if len(args) < 2 {
+		// If a destination is not specified use the base of the source path
 		dst = path.Base(src)
```

Comment:
You're right, that's much cleaner. I thought assigning `dst` twice would be unnecessary, but it really just cluttered up the code with an extra `else`.